assets/sass/patterns/molecules/teaser.scss

```diff
@@ -16,57 +16,84 @@
 .teaser__context_label_list {
   @include nospace();
   position: relative;
-  @include truncate-with-ellipsis();
-
+  display: inline;
   .listing-list--highlights & {
     display: none;
   }
 
 }
 
+.teaser__curation_label_list {
+  @extend .teaser__context_label_list;
+}
+
 .teaser__context_label_item {
-  display: inline;
+  display: inline-block;
   list-style-type: none;
-  @include nospace();
-
-  & .teaser__context_label:after {
-    content: ", ";
-  }
-
-  &:last-child {
-    & .teaser__context_label:after {
-      content: "";
-    }
-  }
+  //@include nospace();
+  margin-bottom: 0.75rem;
+
+  //& .teaser__context_label:after {
+  //  content: ", ";
+  //}
+  //
+  //&:last-child {
+  //  & .teaser__context_label:after {
+  //    content: "";
+  //  }
+  //}
+}
 
+.teaser__curation_label_item {
+  @extend .teaser__context_label_item;
 }
 
 .teaser__context_label {
   // content-label rather than subject-label as not it's not always subjects in here
   @include label-content-typeg();
+  @include font-size-and-vertical-height(12);
 }
 
 // Extra specificity to override normalised style
 .teaser .teaser__context_label_link {
   text-decoration: none;
 
   .teaser__context_label {
+    background-color: #ebf2fa;
```

Comment:
Class attributes should be in alphabetical order

Reply:
The class attributes are ordered alphabetically.